pkg/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -14,6 +14,11 @@
     <PackageTargetRuntime></PackageTargetRuntime>
   </PropertyGroup>
   
+  <!-- The package references are used to generate a runtimes.json for the meta-package-->
+  <ItemGroup>
+    <ProjectReference Include="TargetSpecific\Microsoft.DotNet.ILCompiler.pkgproj" />
```

Comment:
Is the runtimes.json going to have the right content, even though we are referencing just one RID-specific package during the build?

Reply:
Yes. The references are traversed first it seems and the content added to the metapackage later, according to the PackageTargetRuntime that we've set in the TargetSpecific pkgproj.